openvino/CMakeLists.txt

```diff
@@ -15,5 +15,7 @@
 # ******************************************************************************
 
 add_subdirectory(itt)
+add_subdirectory(cc)
 
 openvino_developer_export_targets(openvino::itt)
+openvino_developer_export_targets(openvino::cc)
```

Comment:
we can combine 2 `openvino_developer_export_targets` into a single call.

Reply:
fixed